OneCommunityGlobal/HighestGoodNetworkApp
# Yongjian add to projects report copy functionalities

# Description
<img width="765" alt="Screenshot 2023-09-19 at 5 08 15 PM" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/46613683/0b0010a0-6b82-4e7a-8324-f881ff3face7">

## Main changes explained:
Aligned green dots under "Active" label and made ID numbers able to be copied to clipboard upon hovering cursor over them. Made WBS names clickable and take users to their corresponding task list.

## How to test:
1. check into current branch
2. do `npm run start:local` to run this PR locally
3. log as admin user
4. go to Dashboard→ Reports -> Reports -> Projects -> "AAA USE THIS PROJECT" -> click on the WBS to take to task list
5. verify green dots under "Active" label are aligned and ID numbers are able to be copied to clipboard.

## Screenshots or videos of changes:
<img width="1701" alt="Screenshot 2023-09-19 at 5 05 40 PM" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/46613683/39d3184f-ec07-43a6-9570-f21fc0f6f4f8">
<img width="1701" alt="Screenshot 2023-09-19 at 5 05 46 PM" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/46613683/743838ac-a08f-422b-a411-d98ad36238ec">
<img width="1701" alt="Screenshot 2023-09-19 at 5 06 16 PM" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/46613683/ed1b5e57-14eb-440b-89c3-bf4b93f66b40">

<img width="1701" alt="Screenshot 2023-09-19 at 5 06 20 PM" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/46613683/1df3c923-c8d1-49f9-ae05-b2b034d9c3e9">
<img width="1701" alt="Screenshot 2023-09-19 at 5 06 26 PM" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/46613683/c5412ec5-0ae0-49c8-8fb6-2295594cf68d">
